chocolatey/lektor.nuspec

```diff
@@ -0,0 +1,34 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>lektor</id>
+    <version>3.2.0</version>
+    <packageSourceUrl>https://github.com/lektor/lektor</packageSourceUrl>
+    <owners>Nimish Bongale</owners>
+
+    <title>lektor</title>
+    <authors>lektor</authors>
+    <projectUrl>https://www.getlektor.com/</projectUrl>
+    <!--<iconUrl>http://cdn.rawgit.com/__REPLACE_YOUR_REPO__/master/icons/lektor.png</iconUrl>-->
+    <!-- <copyright>Year Software Vendor</copyright> -->
+    <licenseUrl>https://github.com/lektor/lektor/blob/master/LICENSE</licenseUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    
+    <projectSourceUrl>https://github.com/lektor/lektor</projectSourceUrl>
+    <docsUrl>https://www.getlektor.com/docs/</docsUrl>
+    <!--<mailingListUrl></mailingListUrl>-->
+    <bugTrackerUrl>https://github.com/lektor/lektor/issues</bugTrackerUrl>
+    <tags>python pipenv cms</tags>
+    <summary>lektor</summary>
+    <description>The lektor static file content management system</description>
+    <!-- <releaseNotes>__REPLACE_OR_REMOVE__MarkDown_Okay</releaseNotes> -->
+
+    <dependencies>
+      <dependency id="python"/>
+      <dependency id="imagemagick"/>
```

Comment:
We should also add ffmpeg to here

Reply:
It seemed to work even without it, but I'll go ahead and add it no issues.